src/index.ts

```diff
@@ -1,6 +1,7 @@
 import { config } from 'dotenv';
 import yargs from 'yargs/yargs';
 import { hideBin } from 'yargs/helpers';
+import { logger } from './logger';
```

Comment:
I think having a way to filter logs by class/module/source-file is essential. Is there an easy way to do that with winston?

Reply:
Say more? Is that filtering something you expect/want to do in this repo, or a capability you think is important in logging systems in general? How is that done in pino?

Does having [multiple loggers](https://github.com/winstonjs/winston/blob/master/README.md#working-with-multiple-loggers-in-winston) address that concern?

Reply:
In pino I believe it's done by providing a `source` -- see https://github.com/PhilanthropyDataCommons/service/blob/b04d4178a32fc5a03ffcb56acf066d4c23021549/src/logger.ts#L28

Reply:
Ah, that's helpful - so yes I think we could easily do the same here if we want.

How is the filtering done, though - like how is that capability applied in practice?

Reply:
@bickelj I largely consider that to be noise outside of exceptions :upside_down_face: it isn't currently present, no.